extra/1-radio-stations.js

```diff
@@ -14,7 +14,10 @@
  */
 
 // `getAllFrequencies` goes here
-
+function getAllFrequencies(){
+  let freqArray = [87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108]
```

Comment:
Is there any way you could do this without manually writing the stations one by one? Maybe only defining the start and the end?

Reply:
changed it to a loop